File: scripts/plots.jl
include("../src/sublinearLV.jl")
using DrWatson

p = Dict{Symbol, Any}(:S => 100, :μ => 1, :σ => .1, :λ => 0.1, :T => .1, :K => 50, :symm => true)


plt = LV_plot(p, max_time = 100)
plt = plot(plt, plot_title = savename(p), titlefont=font(10,"Computer Modern"))
png(plt, savename("plots/", p))

p = Dict{Symbol, Any}(:S => 100, :μ => 1, :σ => .1, :λ => 0.1, :T => .01, :K => 50, :symm => false)

plt = LV_plot(p, max_time = 500)
plt = plot(plt, plot_title = savename(p), titlefont=font(10,"Computer Modern"))
png(plt, savename("plots/", p))

File: scripts/plots.py
from sublinearLV import *
%matplotlib inline


### May rescaled


lv = LotkaVolterra(S = 20, mu = 1, sigma = 0.1, l=.1, K = 100, T = 0.01, symm = True)

X0 = np.random.random((5, lv.S))
for i in range(5):
    lv.plot(x0 = X0[i], max_time = 50, dt = .1)



lv = LotkaVolterra(S = 50, mu = 1, sigma = 0.1, l=.1, K = 100, T = 0.01, symm = True)

X0 = np.random.random((5, lv.S))
for i in range(5):
    lv.plot(x0 = X0[i], max_time = 50, dt = .1)

lv = LotkaVolterra(S = 50, mu = 1, sigma = 0.1, l=.1, K = 100, T = 0.01, symm = False)

X0 = np.random.random((5, lv.S))
for i in range(5):
    lv.plot(x0 = X0[i], max_time = 50, dt = .01)


lv = LotkaVolterra(S = 50, mu = 0, sigma = 0, l=.1, K = 100, T = 0.01, symm = False)

X0 = np.random.random((5, lv.S))
for i in range(5):
    lv.plot(x0 = X0[i], max_time = 50, dt = .01)


### not rescaled


lv = LotkaVolterra(S = 5, mu = 1, sigma = 0.1, l=.1, K = 100, T = 0.01, symm = True, scaled = False)

X0 = np.random.random((5, lv.S))
for i in range(5):
    lv.plot(x0 = X0[i], max_time = 50, dt = .01)

lv = LotkaVolterra(S = 50, mu = 1, sigma = 0.1, l=.1, K = 100, T = 0.01, symm = True, scaled = False)

X0 = np.random.random((5, lv.S))
for i in range(5):
    lv.plot(x0 = X0[i], max_time = 50, dt = .01)


# test


lv = LotkaVolterra(S = 100, mu = 1, sigma = 0.1, l=.1, K = 100, T = 0.01, symm = True)
